wamr-compiler/CMakeLists.txt

```diff
@@ -263,15 +269,23 @@ add_library (aotclib ${IWASM_COMPL_SOURCE})
 
 add_executable (wamrc main.c)
 
+if (LLVM_LINK_LLVM_DYLIB)
+  set(WAMRC_LINK_LLVM_LIBS LLVM)
+else()
+  set(WAMRC_LINK_LLVM_LIBS ${LLVM_AVAILABLE_LIBS})
+endif()
+
 if (NOT MSVC)
-  target_link_libraries (wamrc aotclib vmlib LLVMDemangle ${LLVM_AVAILABLE_LIBS} ${lib_ubsan}
+  target_link_libraries (wamrc aotclib vmlib ${WAMRC_LINK_LLVM_LIBS} ${lib_ubsan}
                          -lm -lpthread ${lib_lldb} ${UV_A_LIBS})
   if (MINGW)
-      target_link_libraries (wamrc -lssp -lWs2_32)
+      target_link_libraries (wamrc ssp.a ws2_32)
```

Comment:
Looks good, have you tested building wamrc on `Windows` and `Windows MinGW`?

Reply:
I fixed a compilation error and built a worked wamrc by MSYS2 `mingw-w64-x86_64-gcc` compiler with MSYS2 `mingw-w64-x86_64-llvm` installation.